src/Mvc/src/Microsoft.AspNetCore.Mvc.ViewFeatures/Microsoft.AspNetCore.Mvc.ViewFeatures.csproj

```
@@ -13,13 +13,16 @@ Microsoft.AspNetCore.Mvc.ViewComponent</Description>
     <PackageTags>aspnetcore;aspnetcoremvc</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepositoryRoot)src\Shared\ChunkingCookieManager\**\*.cs" />
```

```suggestion
    <Compile Include="$(MSBuildThisFileDirectory)..\..\..\Shared\ChunkingCookieManager\**\*.cs" />
```

Until MVC converts to the new ref style, RepositoryRoot will point to src/Mvc/